libr/cons/grep.c

```diff
@@ -340,10 +368,10 @@ static char *find_next_intgrep(char *cmd, const char *quotes) {
 			break;
 		}
 		if (p == cmd || *(p - 1) != '\\') {
-			return (char*)p;
+			return (char *)p;
 		}
 		//twiddle unescape
-		memmove (p - 1, p, strlen(p) + 1);
+		memmove (p - 1, p, strlen (p) + 1);
```

Comment:
r_str_cpy does the strlen inside and used memmove. Is p-1 safe? (I think so, just looking for a quick double check after your changes)